.github/workflows/artifacts.yaml

```diff
@@ -175,6 +177,22 @@ jobs:
       #     path: sbom-spdx.json
       #     retention-days: 5
 
+      # TODO: uncomment when the action is working for non ghcr.io pushes. GH Issue: https://github.com/actions/attest-build-provenance/issues/80
+      - name: Generate build provenance attestation
+        - uses: actions/attest-build-provenance@173725a1209d09b31f9d30a3890cf2757ebbff0d # v1.1.2
```

Comment:
Apologies, I messed up the recommendation here :(

Reply:
and me that did not check :/